app/Filament/Resources/WebhookResource.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Filament\Resources\WebhookResource\Pages;
+use App\Models\Webhook;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Table;
+
+class WebhookResource extends Resource
```

Comment:
I don't think it makes sense to have a whole resource for the actual webhook records themselves. I'd say a tab and/or table under that specific Webhook Config. Also you can name Webhook Configs to just "Webhooks".

Reply:
Yeah I was thinking of just a Resource with the Tabs "Webhhooks" and "Logs"